automation/check-patch.e2e-multus-functests.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -xeuE
+
+# This script should be able to execute multus
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-multus-functests.sh
+
+teardown() {
+    rm -rf "${TMP_COMPONENT_PATH}"
+    make cluster-down
+
+}
+
+function __get-tools() {
+    if [ ! -d bin ]; then
+        mkdir bin
```

Comment:
u can simply mkdir -p bin please, and no if needed

Reply:
Done